src/app/components/Field/Field.test.tsx

```diff
@@ -0,0 +1,196 @@
+import userEvent from '@testing-library/user-event'
+
+import { Factory } from '../../../game/services/Factory'
+import { updateField } from '../../../game/reducers/update-field'
+import { stubGame } from '../../../test-utils/stubs/game'
+import { carrot, pumpkin } from '../../../game/cards'
+import * as cards from '../../../game/cards'
+import { isCardId } from '../../../game/types/guards'
+import { playedCropWrapperClassName } from '../PlayedCrop'
+import { cardClassName } from '../Card/CardTemplate'
+
+import { Field, FieldProps, rotationTransform } from '.'
+
+let gameStub = stubGame()
+const opponentPlayerId = Object.keys(gameStub.table.players)[1]
+
+const fieldCrop1 = carrot
+const fieldCrop2 = pumpkin
+
+const cropsStub = [
+  { ...Factory.buildPlayedCrop(fieldCrop1), waterCards: 1 },
+  { ...Factory.buildPlayedCrop(fieldCrop2), waterCards: 3 },
+]
+
+gameStub = updateField(gameStub, gameStub.sessionOwnerPlayerId, {
+  crops: cropsStub,
+})
+
+gameStub = updateField(gameStub, opponentPlayerId, {
+  crops: cropsStub,
+})
+
+const StubField = (overrides: Partial<FieldProps>) => {
+  return (
+    <Field
+      game={gameStub}
+      playerId={gameStub.sessionOwnerPlayerId}
+      {...overrides}
+    />
+  )
+}
+
+describe('Field', () => {
+  test('renders crop cards for self', () => {
+    render(<StubField />)
+
+    for (const crop of cropsStub) {
+      if (!isCardId(crop.id)) {
+        throw new Error()
+      }
+
+      const card = screen
+        .getByText(cards[crop.id].name)
+        .closest(`.${cardClassName}`)
+
+      const { transform } = getComputedStyle(card!)
+      expect(transform).toMatchSnapshot()
+    }
+  })
+
+  test('renders crop cards for opponent upside-down', async () => {
+    render(<StubField playerId={opponentPlayerId} />)
+
+    for (const crop of cropsStub) {
+      if (!isCardId(crop.id)) {
+        throw new Error()
+      }
+
+      const card = screen
+        .getByText(cards[crop.id].name)
+        .closest(`.${playedCropWrapperClassName}`)
+
+      const { transform } = getComputedStyle(card!)
+      expect(transform).toMatchSnapshot()
+      expect(transform).toContain(rotationTransform)
+    }
+  })
+
+  test('clicking a card selects it', async () => {
+    render(<StubField />)
+
+    const playedCrop1 = screen
+      .getByText(fieldCrop1.name)
+      .closest(`.${cardClassName}`)
+
+    const playedCrop1Wrapper = screen
+      .getByText(fieldCrop1.name)
+      .closest(`.${playedCropWrapperClassName}`)
+
+    await userEvent.click(playedCrop1!)
+
+    const { transform: playedCrop1Transform } = getComputedStyle(
+      playedCrop1Wrapper!
+    )
+
+    expect(playedCrop1Transform).toMatchSnapshot()
+
+    for (const crop of cropsStub.slice(1)) {
+      if (!isCardId(crop.id)) {
+        throw new Error()
+      }
+
+      const cardWrapper = screen
+        .getByText(cards[crop.id].name)
+        .closest(`.${playedCropWrapperClassName}`)
```

Comment:
i don't remember the exact accessibility standard, but in the past i've seen visually hidden text added to a selected item to indicate it was selected, which feels applicable here. so you could instead look for ```.getByText(`${cards[crop.id].name}, selected`)``` for a more meaningful "is selected" check (note: the "," is added on purpose to make the voice over give a slight pause)